SROS2_Linux.md

```diff
@@ -130,15 +130,15 @@ First, we'll create an empty keystore on `oldschool`, which is just an empty dir
 
 ```
 ssh oldschool.local
-mkdir -p ~/sros2_demo/demo_keys
+mkdir -p ~/sros2_demo/demo_keys/enclaves/talker_listener
```

Comment:
Ah, having to create this structure isn't ideal, is it :thinking:

Reply:
@kyrofa 

I also thought about it 🤔. I checked what options the cp command has and I couldn't find anything good to use for coping a file to a non-existing file path (if you don't make directory path /.../.enclaves/talker_linstener, you cannot really copy a folder there).